common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2433,20 +2433,24 @@ StScrollBar {
 //
 .user-icon {
   background-size: contain;
-  color: $fg_color;
+  color: $osd_fg_color;
   border-radius: 99px;
+
+  .login-dialog-user-list-item:selected & { color: $selected_fg_color; }
+
   &:hover {
-    color: transparentize($fg_color, 0.3);
+    color: if($variant != 'lighter', lighten($osd_fg_color, 25%), darken($osd_fg_color, 12%));
   }
 
   & StIcon {
-    background-color: transparentize($osd_fg_color, 0.95);
+    background-color: transparentize($osd_fg_color, 0.75);
     border-radius: 99px;
   }
 }
 
 .user-widget.vertical .user-icon {
   icon-size: 128px;
+  color: if($variant != 'lighter', lighten($osd_fg_color, 25%), darken($osd_fg_color, 12%));
```

Comment:
Here again, I think we should keep the unadjusted `$osd_fg_color;` for consistency.

Reply:
Similar comment as the one below re `.user-widget.vertical` elements

Reply:
Please use `color: $osd_fg_color;` here. See my comment for the `.user-widget.vertical .user-widget-label` below.

Reply:
Done